subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/jvm/JavaToolchainFixture.groovy

```diff
@@ -97,7 +97,7 @@ trait JavaToolchainFixture {
      * Returns the Java version from the compiled class bytecode.
      */
     JavaVersion classJavaVersion(File classFile) {
-        assert classFile.exists()
+        assert classFile.exists(), "Class file ${classFile} does not exist"
```

Comment:
You are correct. The comma syntax also works, but I completely forgot that Spock, in case of an error, provides all the info for expressions anyway. So this message here is redundant. Thanks for noticing!